scripts/lanl-builds/cray_nvcc_gcc.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2016-24, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+if [[ $# -lt 3 ]]; then
+  echo
+  echo "You must pass 3 arguments to the script (in this order): "
+  echo "   1) compiler version number for nvcc"
+  echo "   2) CUDA compute architecture (number only, not 'sm_90' for example)"
+  echo "   3) compiler version number for gcc"
+  echo
+  echo "For example: "
+  echo "    cray_nvcc_gcc.sh 12.5 90 12.3.0"
+  exit
+fi
+
+COMP_NVCC_VER=$1
+COMP_ARCH=$2
+COMP_GCC_VER=$3
```

Comment:
Looks like these values aren't used anymore? I think we keep most of our stuff in the script so its more configurable.

Reply:
Ok moved some options back into the script. If there are any more to move, please enumerate them